add-on/panels/js/panel.jsx

```diff
@@ -19,31 +19,53 @@ loop.panel = (function(_, mozL10n) {
   var GettingStartedView = React.createClass({
     mixins: [sharedMixins.WindowCloseMixin],
 
+    propTypes: {
+      gettingStartedContainer: React.PropTypes.string.isRequired
```

Comment:
This wants to be something that's not defined based on enclosing components, as this violates modularity and makes the code harder to reason about.  Maybe it wants to be a boolean "displayRoomListContent" or some such...